infrastructure/code/code_html.go

```diff
@@ -246,6 +246,26 @@ func getRepoName(commitURL string) string {
 	return tabTitle
 }
 
+func formatExpirationDate(expiration string) string {
+	if expiration == "" {
+		return "No expiration"
+	}
+	parsedDate, err := time.Parse(time.RFC3339, expiration)
+	if err != nil {
+		return expiration // Original string if parsing fails
+	}
+
+	// Calculate the difference in days
+	daysRemaining := int(parsedDate.Sub(time.Now()).Hours() / 24)
```

Comment:
Have you tried doing what the GitHub action suggests? Using `time.Until`

Reply:
I just saw the linter recommendation 💪 
I fixed it 🙌